scripts/generate_download_page.sh

```diff
@@ -115,7 +115,7 @@ files=($(${SSH} "find ${src_dir} -maxdepth 1 -type f -not -name 'config.php' -no
 # Loop through files, generate a table entry for each one
 for file in "${files[@]}"; do
     filename=$(basename $file)
-    size=($(${SSH} "du -h '${src_dir}/$file' | awk '{print $1}'"))
+    size=($(${SSH} "du -h "$file" | awk '{print $1}'"))
```

Comment:
it should be single quotes around `$file`

Reply:
corrected